timed/subscription/tests/test_subscription_project.py

```diff
@@ -64,3 +65,20 @@ def test_subscription_project_detail(auth_client):
     assert res.status_code == HTTP_200_OK
     json = res.json()
     assert json["data"]["id"] == str(project.id)
+
+
+def test_subscription_project_list_customer(auth_client):
+    customer = CustomerFactory.create()
+    ProjectFactory.create(customer=customer, customer_visible=True)
+    ProjectFactory.create_batch(4, customer_visible=True)
+
+    user = auth_client.user
+    CustomerAssigneeFactory.create(user=user, customer=customer, is_customer=True)
+
+    url = reverse("subscription-project-list")
+
+    response = auth_client.get(url)
+    assert response.status_code == HTTP_200_OK
+
+    json = response.json()
+    assert len(json["data"]) == 1
```

Comment:
I suggest checking not only the length, but also the content of the returned data (ID of the returned project should be the exact one that we're allowed to see)